protocols/plaintext/src/pb/structs.rs

```diff
@@ -0,0 +1,278 @@
+// This file is generated by rust-protobuf 2.3.0. Do not edit
```

Comment:
Protobuf version will be updated once plaintext moves to the consolidated scripts. (Seem comment above)